pkg/cluster/eventbus.go

```diff
@@ -29,34 +32,66 @@ func newEventbusService(cc *raw_client.Conn, svc NamespaceService) EventbusServi
 }
 
 func (es *eventbusService) GetSystemEventbusByName(ctx context.Context, name string) (*meta.Eventbus, error) {
+	v, exist := es.cache.Load(name)
+	if exist {
+		return v.(*meta.Eventbus), nil
+	}
+
 	pb, err := es.nsSvc.GetSystemNamespace(ctx)
 	if err != nil {
 		return nil, err
 	}
-	return es.client.GetEventbusWithHumanFriendly(ctx, &ctrlpb.GetEventbusWithHumanFriendlyRequest{
+	eb, err := es.client.GetEventbusWithHumanFriendly(ctx, &ctrlpb.GetEventbusWithHumanFriendlyRequest{
 		NamespaceId:  pb.Id,
 		EventbusName: name,
 	})
+	if err != nil {
+		return nil, err
+	}
+	es.cache.Store(name, eb)
+	return eb, nil
 }
 
 func (es *eventbusService) GetEventbus(ctx context.Context, id uint64) (*meta.Eventbus, error) {
-	return es.client.GetEventbus(ctx, wrapperspb.UInt64(id))
+	v, exist := es.cache.Load(id)
+	if exist {
+		return v.(*meta.Eventbus), nil
+	}
+	eb, err := es.client.GetEventbus(ctx, wrapperspb.UInt64(id))
+	if err != nil {
+		return nil, err
+	}
+	es.cache.Store(id, eb)
```

Comment:
when the eventbus is deleted the cache need to be cleaned, so if uses id it can't cache?